.gitignore

```diff
@@ -1 +1,2 @@
 .build
+*.swp
```

Comment:
I always neglect adding this because I mercilessly disable the swap file in vim :)